django/db/models/base.py

```diff
@@ -335,12 +338,9 @@ def make_foreign_order_accessors(field, model, cls):
                     'set_%s_order' % cls.__name__.lower(),
                     curry(method_set_order, cls)
                 )
-            add_lazy_relation(
-                cls,
-                opts.order_with_respect_to,
-                opts.order_with_respect_to.rel.to,
-                make_foreign_order_accessors
-            )
+            lazy_related_operation(make_foreign_order_accessors,
```

Comment:
Yes I do. A couple reasons:
* It's easier not to have to indent a line some non-multiple of 4 spaces
* No need to adjust indentation if positioning of the opening ( changes
* Less whitespace on the left hand side / longer lines